doc-content/enterprise-only/bpmn/bpmn-end-events-ref.adoc

```diff
@@ -74,4 +74,4 @@ The escalation end event finishes the incoming workflow, which means consumes th
 [[_terminate_end_event]]
 .Terminate
 
-The terminate end event finishes all execution flows in the specified process instance. Activities being executed are canceled. If a terminate end event is reached in a subprocess, the entire process instance is terminated.
+The terminate end event finishes all execution flows in the specified process instance. Activities being executed are canceled. The subprocess instance is terminated if a terminate end event is reached in a subprocess.
```

Comment:
Minor suggestion - Can it be simplified further?

s/The subprocess instance is terminated if a terminate end event is reached in a subprocess./The process instance stops immediately if a terminate end event is reached.

Reply:
*or subprocess

Reply:
How about changing to:

The subprocess instance is terminated if it is reached a terminate end event. //I am not sure about the word 'stop' since it has two meanings: instance is stopped for now and can be started again, or it has stopped.

Reply:
@hmanwani-rh  Thanks for pointing that out about "stop". Last suggestion :) can we avoid passive voice or do you think it helps clarity in this case?

"The subprocess instance terminates if it reaches a terminate end event." or "The subprocess instance is terminated if a terminate end event is reached" (going back to your original :) )

Reply:
@csherrar I would like to go with the first one (without passive voice):
The subprocess instance terminates if it reaches a terminate end event. 

thanks for coming up with the suggestion. :)